test/KregielniaTests.cpp

```diff
@@ -9,3 +9,38 @@ TEST_F(KregielniaTests, assertThatTravisCiBuildStatusIsPass)
 {
     ASSERT_TRUE(true);
 }
+
+TEST_F(KregielniaTests, assertThatCanCreateTestFiles)
+{
+    bool result = true;
+    fs::path p1;
+    p1 /= Helper::INPUT_FILES_DIRECTORY;
+    // GIVEN
+    result &= not fs::is_directory(p1);
```

Comment:
There should be immediate check:
ASSERT_FALSE(fs::is_directory(p1));
The same for lines 23, 24 and in the other test case.
Then the variable result is not needed.